test/visualization/test_visualization.py

```diff
@@ -87,3 +87,13 @@ def test_credible_intervals():
 
 def test_model_probabilities():
     pyabc.visualization.plot_model_probabilities(histories[0])
+
+
+def test_data_plot():
```

Comment:
this would be enough tests for me at the moment (in particular, since my own tests of visualization routines also just execute code ^^)